source/scenario/newmission3task3.vwf.yaml

```diff
@@ -145,13 +145,26 @@ children:
               - enabled
               - true
 
-        highlightEndTile:
+        highlightEndTile3_3:
           triggerCondition:
           - onScenarioStart:
           actions:
           - callOutObjective:
             - [ -17, 26 ]
 
+        failOnMissingBlock_3e:
+          group: successOrFailure
+          priority: 3
+          triggerCondition:
+          - or:
+            - onBlockMissing:
+              - rover
+              - controls_if_nomut
+          actions:
+          - scenarioFailure:
+            - incomplete
+            - "You must use an if block."
```

Comment:
Also, on all of these we should think about the error message.  Is there a more generic error message we can give (or else maybe a more specific one)?  I don't want to tell them what the failure check is so that they can find another way to work around it, I want them to have to solve the problem properly.

Let's sit down together and talk through which missions these are...